kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -50,9 +49,12 @@ def create_connector(
         """
         config_json = kafka_connect_config.dict(exclude_none=True)
         connect_data = {"name": connector_name, "config": config_json}
-        response = httpx.post(
+
+        client = httpx.AsyncClient()
```

Comment:
you could initialize client as an attribute of the Kafka Connect wrapper and re-use it in all the methods

Reply:
duplicate of https://github.com/bakdata/kpops/pull/301#discussion_r1279196833